src/redux/sandbox.js

```diff
@@ -0,0 +1,245 @@
+/* eslint-disable */
```

Comment:
- [ ] Kindly avoid disabling eslint rule. It's best to fix the issue in the code.